helper.py

```diff
@@ -43,21 +34,17 @@ def upload_template_python_script():
 
 if __name__ == "__main__":
     upload_template_python_script()
-    Stack = stack.Stack(parameter)
-    status = Stack.create_update_stack()
+    Stack = stack.Stack(STACK_NAME, TEMPLATE_URL, DATABASE_NAME, BUCKET_NAME, CRAWLER_NAME, JOB_NAME, DATA_BUCKET)
```

Comment:
variables and instance naming should be lowercase.
```suggestion
    stack_obj = stack.Stack(STACK_NAME, TEMPLATE_URL, DATABASE_NAME, BUCKET_NAME, CRAWLER_NAME, JOB_NAME, DATA_BUCKET)
```